binding-mruby/binding-mruby.cpp

```diff
@@ -115,24 +115,9 @@ static void mrbBindingInit(mrb_state *mrb)
 	mrb_load_irep(mrb, mrbModuleRPG);
 
 	/* Load global constants */
-	bool has_debug = false;
-	bool has_test = false;
-	bool has_btest = false;
-	for (int i = 1; i < shState->rtData().argc; i++)
-	{
-		const std::string arg = shState->rtData().argv[i];
-		if ("debug" == arg)
-			has_debug = true;
-		else if ("test" == arg)
-			has_test = true;
-		else if ("btest" == arg)
-			has_btest = true;
-	}
-
 	mrb_define_global_const(mrb, "MKXP", mrb_true_value());
-	mrb_define_global_const(mrb, "DEBUG", has_debug ? mrb_true_value() : mrb_false_value());
-	mrb_define_global_const(mrb, "TEST", has_test ? mrb_true_value() : mrb_false_value());
-	mrb_define_global_const(mrb, "BTEST", has_btest ? mrb_true_value() : mrb_false_value());
+	mrb_define_global_const(mrb, "DEBUG", mrb_bool_value(shState->config().editor.debug));
```

Comment:
Might want to set `TEST` here too according to `rgssVer` like you did in MRI, just for consistency.